kickstart/provision.erb

```diff
@@ -183,12 +183,14 @@ echo "updating system time"
 echo 'proxy = <%= proxy_uri %>' >> /etc/yum.conf
 <% end -%>
 
+<% unless @host.param_false?('update_base_packages') -%>
```

Comment:
There's already a `package_upgrade` parameter used in cloudinit/userdata_cloudinit.erb, so I'd suggest renaming this parameter. Please also add it to the docs at the top of the file.